### DISCUSSION TITLE
Config Error // Incorrect print size due to wrong wkhtmltopdf

### DISCUSSION THREAD

**Discussion by LeHarry** [frappe/print_designer#172] [Q&A] [answered]
=> input

![grafik](https://github.com/frappe/print_designer/assets/160015230/fae15c68-3b10-4a7d-9988-d9a3dfbc565d)

=> output

![grafik](https://github.com/frappe/print_designer/assets/160015230/68f479b4-cc4f-4430-9efd-44d789c63323)

**Comment by maharshivpatel** [comment]
@LeHarry install correct version of wkhtmltopdf https://github.com/frappe/print_designer?tab=readme-ov-file#faq

**Reply by LeHarry** [reply]
```
$ wkhtmltopdf --version
  wkhtmltopdf 0.12.6
```

**Reply by maharshivpatel** [reply]
it should be **( qt patched )** `wkhtmltopdf 0.12.6 (with patched qt)`

**Reply by LeHarry** [reply]
The documentation is very vague about the patched qt update. How does one get and replace the module that comes with the standard installation with `wkhtmltopdf 0.12.6 (with patched qt)` on a standard Debian 12 OS?

**Reply by maharshivpatel** [reply]
https://github.com/frappe/print_designer/issues/170#issuecomment-1956109192

**Comment by LeHarry** [comment, marked as answer]
Solution:

**Download the latest `wkhtmltopdf`**
I got mine for Debian 12 from: https://github.com/wkhtmltopdf/packaging/releases
Make sure to get the right version for your OS. Copy the link address. You can download the link using e.g. wget
`$ wget %insert_link_to_your_file%`

**Install:**
`$ sudo apt install ./wkhtmltox_0.12.6.1-3.bookworm_amd64.deb`

**Check version:**
`$ wkhtmltopdf --version`
`  wkhtmltopdf 0.12.6.1 (with patched qt)`
